app/views/photos/index.html.haml

```diff
@@ -0,0 +1,27 @@
+- if @photos.any?
+  - @photos.each do |photo|
+    .row
+      .ul.list-group
+        .li.list-group-item
+          %p.small= "#{time_ago_in_words(photo.created_at)} ago"
+          %p= "by #{photo.username}"
+        .li.list-group-item
+          = image_tag photo.image.url, class: 'img-rounded'
+        .li.list-group-item
+          = link_to icon('heart-o'), photo_likes_path(photo), method: :post, class: 'like-link'
+          %span.likes-count photo.likes.count
+        .li.list-group-item
+          - if current_user && current_user.owns?(photo)
+            = link_to 'Delete photo', photo_path(photo), method: :delete
+        .li.list-group-item
+          .form-group
+            %input{:type=>"text"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace